src/value/owned.rs

```diff
@@ -357,110 +350,169 @@ impl Element for OwnedElement {
 
 #[cfg(test)]
 mod value_tests {
-    use crate::result::IonResult;
```

Comment:
We don't leverage the `?` operator in these tests, so I didn't bother keeping this.